dev-environment.yml

```diff
@@ -23,7 +23,7 @@ channels:
 dependencies:
```

Comment:
The commit message of this commit has a typo in the title.

I would recommend just squashing this commit into the previous one.